src/Dialogs/Frequency/dlgUserFrequencyList.cpp

```
@@ -0,0 +1,316 @@
+  form->SetModalResult(mrOK);
+}
+
+void UserFrequencyListWidget::UpdateFile()
+{
+  const auto path = LocalPath(file_name);
+
+  FileOutputStream file(path);
+  BufferedOutputStream writer(file);
+
+  for (const auto &item : list)
+  {
```

Cppcheck (2.20.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)